src/Geopilot.Frontend/vite.config.js

```diff
@@ -1,8 +1,29 @@
 import { fileURLToPath, URL } from "node:url";
+import process from "node:process";
 
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import viteTsconfigPaths from "vite-tsconfig-paths";
+import fs from "fs";
+import path from "path";
+
+const baseFolder =
+  process.env.APPDATA !== undefined && process.env.APPDATA !== ""
+    ? `${process.env.APPDATA}/ASP.NET/https`
+    : `${process.env.HOME}/.aspnet/https`;
+
+const certificateArg = process.argv.map(arg => arg.match(/--name=(?<value>.+)/i)).filter(Boolean)[0];
+const certificateName = certificateArg ? certificateArg.groups.value : process.env.npm_package_name;
+
+if (!certificateName) {
+  console.error(
+    "Invalid certificate name. Run this script in the context of an npm/yarn script or pass --name=<<app>> explicitly.",
+  );
+  process.exit(-1);
+}
+
+const certFilePath = path.join(baseFolder, `${certificateName}.pem`);
+const keyFilePath = path.join(baseFolder, `${certificateName}.key`);
```

Comment:
Q: Dasselbe wird in ähnlich in [aspnetcore-https.js](https://github.com/GeoWerkstatt/geopilot/blob/main/src/Geopilot.Frontend/aspnetcore-https.js) für dotnet gemacht, braucht es das File [aspnetcore-https.js](https://github.com/GeoWerkstatt/geopilot/blob/main/src/Geopilot.Frontend/aspnetcore-https.js) überhaupt noch?

Reply:
Der code evaluiert lediglich wo unter Unix/Windows die Zertifikate liegen sollten. Der code kommt aus dem default Vite/React Template. 
In aspnetcore-https.js wird im npm-Skript `predev` in diesen Pfad exportiert. In vite.config.js entsprechend angezogen. Diese Skripte werden durch `npm run dev` nacheinander aufgerufen. Den code könnte man sicherlich vereinfachen, sofern man bereit ist vom Template abzuweichen. Ein export nach ./certs könnte auch Sinnvoll sein, jedoch bin ich nicht sicher, aus welchen Directories die skripts gestartet werden, um den korrekten relativen Pfad zu erwischen.